package-lock.json

```diff
@@ -1,7410 +1,8 @@
 {
   "name": "@evolv-integrations/monorepo",
```

Comment:
You've run npm install with a different node version
Run nvm use before you do npm install (and make sure you install the version it tells you to)

Reply:
done